src/components/icons/languages/figma.tsx

```diff
@@ -0,0 +1,204 @@
+export function figmaIcon({ className = '', ...props }) {
```

Comment:
@camiloguz23 the components are PascalCase

Reply:
done